start.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# If there is some public key in keys folder
+# then it copies its contain in authorized_keys file
+if [ "$(ls -A /git-server/keys/)" ]; then
+  echo "Found key"
+  cd /home/git
+  cat /git-server/keys/*.pub > .ssh/authorized_keys
```

Comment:
Restrict what the user can do:
```suggestion
  sed 's/^/restrict /' /git-server/keys/*.pub > .ssh/authorized_keys
```